### DISCUSSION TITLE
просьба-пожелание (плагин картинок)

### DISCUSSION THREAD

**Discussion by jorjik** [maxsite/cms#438] [General]
есть просьба: раньше я сидел на младшей версии, а потом обновился на старшую, где в шаблоне добавили картинки - смысл в том, что забивать их в старые посты мне не охота; можно ли создать какай-то плагин, кот. будет генерировать абстрактное изображение? (типа узор, так ещё делают для аватарок) *мне бы подошло

**Comment by maxsite** [comment]
Для этого плагин не нужен, поскольку всё и так работает. В info-top-файлах, там где происходит вывод миниатюры используется функция thumb_generate() для генерации заглушки, если не указано поле изображения записи.

**Reply by jorjik** [reply]
это я заметил, я бы хотел что бы был вариант генерации какого-то рандомного бэкграунд ( пример http://rndimg.com/ )

**Reply by maxsite** [reply]
Составьте ТЗ, бюджет, может кто-то и возьмётся за работу.
